generators/entity/index.js

```diff
@@ -232,6 +231,13 @@ class EntityGenerator extends BaseBlueprintGenerator {
               ? ''
               : this.entityConfig.microserviceName;
           }
+
+          if (this.jhipsterConfig.applications && !this.entityConfig.skipClient) {
+            const remoteConfig = this.jhipsterConfig.applications[this.entityConfig.microserviceName];
+            if (remoteConfig.clientFramework === 'vue') {
```

Comment:
use of constant?

Reply:
Temporary, will drop it soon, once angular menu moves to the microservice side.
I will create the PR removing instead of creating the PR with the constant.